### ISSUE TITLE
Attendee/Ticket Holder - Account - Edit Account Page

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#57]
## Background 
A user needs the ability to add contact information, add a profile picture, change passwords etc. 

Latest Mocks are kept in the sketch file in this [folder](https://www.dropbox.com/sh/gt7c74ndhk7me5w/AADIxnNPe39Xbq3HJUCXfb9xa?dl=0). Please doublecheck that this screen has not changed before working ticket. Latest screens are in the production ready tab. 

## Scenario: Editing Account info
- [x] When I am directed to Account edit
- [x] Then I see
	- [x] A Link to change my profile photo
	- [x] Editable fields for:
		- [x] First Name
		- [x] Last Name
		- [x] Phone/Mobile
		- [x] Email
		- [x] Password
	- [x] A link to Sign Out

## Mock
<img width="220" alt="slack_-_revelers" src="https://user-images.githubusercontent.com/6877299/43958306-837cc3ea-9c70-11e8-837d-182a720be213.png">

**Comment by brittanygay** [comment]
Picking this up to work on once #48 is complete

**Comment by brittanygay** [comment]
Marking this ticket blocked until #48 is complete. Pulling in another ticket from the backlog to work on in the meantime.

**Comment by iamjoshfrank** [comment]
#48 is now in QA, so this ticket is no longer blocked, yeah?

**Comment by brittanygay** [comment]
This is now unblocked. Pulling this in to start working now!

**Comment by brittanygay** [comment]
Ticket is in review and assigned to @daybreaker to check out the few items I have listed in my PR: https://github.com/revelrylabs/bn-mobile-react/pull/78

Once those items are complete and assign back to myself and get a design review before merging.

**Comment by daybreaker** [comment]
@brittanygay back to you - the onPress on the "< Settings" has a black background when you press it. Might need to change that? Or is that by design?

**Comment by brittanygay** [comment]
Thanks @daybreaker! Just updated the underlay color so it's not black. Assigning to Blaze for a design review and getting this merged!

**Comment by brittanygay** [comment]
PR is now merged. This will be ready for QA once TestFlight is updated CC @daybreaker

**Comment by daybreaker** [comment]
Passes on iphone, but on android:


![screenshot_1535662082](https://user-images.githubusercontent.com/12603/44882666-e357c300-ac79-11e8-91e5-824f8050e272.png)

**Comment by iamjoshfrank** [comment]
Marking `blocked` for myself so that I can split out Android bug into separate issue.

**Comment by zekewell** [comment]
Passes on iphone.

**Comment by iamjoshfrank** [comment]
Opened #85 for Android bug. Closing.
